### DISCUSSION TITLE
Dynamic filter values

### DISCUSSION THREAD

**Discussion by ars3nicum** [Budibase/budibase#9901] [Help] [answered]
Is it possible somehow to retrieve parameters values which are set in dynamic filter? I want to make a server side export to CSV of the filtered table data. If I do it by built-in 'export data' action, I have to load all the rows to dataprovider, but since there are a lot of rows, the page loading very slow.

**Comment by ars3nicum** [comment, marked as answer]
I realised how to solve it. I made custom filter form and pass its values to a server. Topic can be closed. Thank you!

**Comment by ConorWebb96** [comment]
Thats great to hear! Glad it has been resolved!
